build/bin/sage-logger

```diff
@@ -89,8 +89,9 @@ else
     # Redirect stdout and stderr to a subprocess running tee.
     # We trap SIGINT such that SIGINT interrupts the main process being
     # run, not the logging.
+
     ( exec 2>&1; eval "$cmd" ) | \
-        ( trap '' SIGINT; tee -a "$logfile" | $SED )
+        ( trap '' SIGINT; if [ -n "$GITHUB_ACTIONS" -a -n "$prefix" ]; then echo "::group::${logname}"; fi; tee -a "$logfile" | $SED; if [ -n "$GITHUB_ACTIONS" -a -n "$prefix" ]; then echo "::endgroup::"; fi )
```

Comment:
Any reason not to use the `ci` env variable?

Reply:
Instead of$GITHUB_ACTIONS use $CI

Reply:
these output markers are specific to GitHub Actions, no?

Reply:
Also true...